business/istio_config.go

```diff
@@ -401,7 +403,12 @@ func (in *IstioConfigService) GetIstioConfigList(criteria IstioConfigCriteria) (
 // - "namespace": 		namespace where configuration is stored
 // - "objectType":		type of the configuration
 // - "object":			name of the configuration
-func (in *IstioConfigService) GetIstioConfigDetails(namespace, objectType, object string) (models.IstioConfigDetails, error) {
+func (in *IstioConfigService) GetIstioConfigDetails(ctx context.Context, namespace, objectType, object string) (models.IstioConfigDetails, error) {
+	if config.Get().Server.Observability.Tracing.Enabled {
+		var span trace.Span
+		ctx, span = otel.Tracer(observability.TracerName).Start(ctx, "GetIstioConfigDetails")
```

Comment:
In all these `otel.Tracer.Start` calls, I see the name of the function passed in to label the trace (here is `GetIstioConfigDetails`). 

Should these strings have additional context? Because what if other packages have the same (or very similar) names? Down below I see `getPermissions` and that seems like a name that might be identical to other package functions.

So should we have a convention to prefix the function name with the package name? i.e.

```suggestion
		ctx, span = otel.Tracer(observability.TracerName).Start(ctx, "business.GetIstioConfigDetails")
```

Reply:
That sounds good to me. It's also possible to set any additional attributes on the trace for more context. This can be stuff like the logged in user, request id, etc. I wasn't entirely sure what would be useful so didn't add any of it in for now but it can be added later. Prefixing with the package name seems like a good convention to start.

Reply:
Instead of prefixing the function name, another option would be to include the package as an attribute/tag on the trace. This makes it less obvious when glancing at the span which package the function came from but might allow for better filtering when querying the spans to say something like "give me all spans for this package". I don't see a way to easily search/filter by span name in the jaeger UI.

Reply:
I don't care as much about how its implemented - just that SOMETHING can discern the traces alittle better - having the package name somewhere is good.